README.md

```diff
@@ -15,18 +15,14 @@ Internet Information Services (IIS) for Windows® Server is a flexible, secure a
 
 ## How to use this image?
 ### Create a Dockerfile with your website
-```
+```Dockerfile
 FROM microsoft/iis
 
-RUN mkdir C:\site
-
-RUN powershell -NoProfile -Command \
-    Import-module IISAdministration; \
-    New-IISSite -Name "Site" -PhysicalPath C:\site -BindingInformation "*:8000:"
+RUN powershell -NoProfile -Command Remove-Item -Recurse C:\inetpub\wwwroot\*
 
-EXPOSE 8000
+WORKDIR /inetpub/wwwroot
 
-ADD content/ /site
+COPY content/ .
```

Comment:
I don't see a delete command, will this leave the default content inside of wwwroot?

i.e. iisstart.png, iisstart.htm.

Reply:
I think you missed it. See L21

Reply:
I did miss it. For some reason I was looking for a docker DELETE command, which I don't think exists.